mirrord/agent/src/steal/ip_tables/output.rs

```diff
@@ -23,18 +21,13 @@ where
     pub fn create(ipt: Arc<IPT>, chain_name: String) -> Result<Self> {
         let managed = IPTableChain::create(ipt, chain_name)?;
 
-        let gid = getgid();
-        managed
-            .add_rule(&format!("-m owner --gid-owner {gid} -p tcp -j RETURN"))
-            .inspect_err(|_| {
-                warn!("Unable to create iptable rule with \"--gid-owner {gid}\" filter")
-            })?;
+        managed.add_rule("! -o lo -j RETURN")?;
```

Comment:
This change combined with changes to `add_redirect`/`remove_redirect` are a noop, right?

Reply:
Yes this should result in same result